paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -47,10 +49,17 @@ class PaparazziPlugin : Plugin<Project> {
         it.dependsOn(variant.mergeResourcesProvider)
       }
 
-      project.tasks.named("test${variant.unitTestVariant.name.capitalize()}")
-          .configure {
-            it.dependsOn(writeResourcesTask)
-          }
+      val testVariantSlug = variant.unitTestVariant.name.capitalize()
```

Comment:
Pass `Locale.US`, although I believe the method is still experimental in 1.3.x

Reply:
https://github.com/cashapp/paparazzi/pull/126